deploy/helm/opsbot/templates/deployment.yaml

```diff
@@ -48,9 +52,9 @@ spec:
             - name: plugins
               mountPath: /additional_plugins
             {{- end }}
-            {{- range .Values.extraVolumes }}
```

Comment:
`extraVolumes` should be kept as an option

Reply:
Reverted changes to extraVolumes 👍